.spectral.js

```diff
@@ -0,0 +1,55 @@
+const ibmCloudValidationRules = require('@ibm-cloud/openapi-ruleset');
+const { enumCaseConvention, operationIdCaseConvention } = require('@ibm-cloud/openapi-ruleset/src/functions');
+const { schemas } = require('@ibm-cloud/openapi-ruleset/src/collections');
+
+module.exports = {
+  extends: ibmCloudValidationRules,
+  rules: {
+    'enum-case-convention': {
+      description: 'enum names must follow pascal case',
+      message: '{{error}}',
+      resolved: true,
+      given: schemas,
+      severity: 'error',
+      then: {
+        function: enumCaseConvention,
+        functionOptions: {
+          type: 'pascal'
+        }
+      }
+    },
+    'operation-id-case-convention': {
+      description: 'Operation ids must be pascal case',
+      message: '{{error}}',
+      resolved: true,
+      given: schemas,
+      severity: 'error',
+      then: {
+        function: enumCaseConvention,
+        functionOptions: {
+          type: 'pascal'
+        }
+      }
+    },
+    'major-version-in-path': 'off',
+    'string-boundary': 'off',
+    'array-boundary': 'off',
+    'schema-description': 'off',
+    'inline-property-schema': 'off',
+    'duplicate-path-parameter': 'off',
```

Comment:
:question:  Why it this off?

Reply:
It's not about a duplicate parameter in one path but is about having multiple paths with the same parameter, and it suggests to create a path section to define it only once. I'm going to do that in the next PR.